src/read_pool.rs

```diff
@@ -299,6 +299,7 @@ mod tests {
 
     impl FlowStatsReporter for DummyReporter {
         fn report_read_stats(&self, _read_stats: ReadStats) {}
+        fn report_write_stats(&self, _write_stats: WriteStats) {}
```

Comment:
I noticed that `FlowStatsReporter` will report write stats while the file was still named as `read_pool.rs`.

Reply:
I did not find a more suitable place, do you have any good suggestions